docs/aca/05-aca-dapr-pubsubapi/index.md

```diff
@@ -587,10 +673,22 @@ az containerapp revision restart `
 --resource-group $RESOURCE_GROUP `
 --name $BACKEND_SVC_NAME  `
 --revision $REVISION_NAME
+
+$REVISION_NAME = (az containerapp revision list `
+        --name $BACKEND_API_NAME  `
+        --resource-group $RESOURCE_GROUP `
+        --query [0].name)
+
+##Restart revision by name                            
+az containerapp revision restart `
+--resource-group $RESOURCE_GROUP `
+--name $BACKEND_API_NAME  `
+--revision $REVISION_NAME
+
 ```
 
-With this in place, you should be able to test the 3 services end to end and should receive a notification email to the task assignee email, the email will look like the below
+With this in place, you should be able to test the 3 services end to end and should receive a notification email to the task assignee email, the email will look like the below. Note that if you opted not to activate the SendGrid then you won't receive the email. In this case its enough to test for adding a new task and making sure the application still functions properly.
```

Comment:
Note that if you opted not to activate the SendGrid then you won't receive the email. In this case you can get the backend processor logs using the command below, you should see an information log with the following template `Started processing message with Task Name '<Your task name is displayed here>'"`
```Powershell
az containerapp logs show `
-n $BACKEND_SVC_NAME `
-g $RESOURCE_GROUP
```